.gitignore

```diff
@@ -115,6 +115,20 @@ open-metadata-implementation/user-interfaces/presentation-server/nodejs/yarn.loc
 open-metadata-implementation/user-interfaces/presentation-server/nodejs/node/**
 open-metadata-implementation/user-interfaces/presentation-server/nodejs/.env
 
+open-metadata-implementation/user-interfaces/presentation-server/cra-client/node_modules
```

Comment:
Quite a number of these files are already excluded by pattern (build, .vscode, *.log). Others may be included by minor modification to the existing patterns (package-lock?)

Not critical for this pr, so could be in a later one, though I can take a look to suggest more specifics if I get the overall (dev - not maven!)  working for the PR in my environment

Reply:
This makes sense, but I like your suggestion for leaving it to a later PR. Perhaps we can discuss in our meeting next week

Reply:
#3575 opened to track

Reply:
Addressed in #3577